types/block.go

```diff
@@ -436,7 +437,8 @@ func (h *Header) Populate(
 	version version.Consensus, chainID string,
 	timestamp time.Time, lastBlockID BlockID,
 	valHash, nextValHash []byte,
-	consensusHash, appHash, resultsHash []byte,
+	consensusHash, nextConsensusHash []byte,
```

Comment:
above you pass the same value for current and next consensus hash, is that correct?